e2e/samples/pipelines/test.haystack-pipeline.yml

```diff
@@ -0,0 +1,40 @@
+version: ignore
+
+components:
+  - name: Reader
+    type: FARMReader
+    params:
+      no_ans_boost: -10
+      model_name_or_path: deepset/bert-medium-squad2-distilled
+      num_processes: 0
+  - name: Retriever
+    type: BM25Retriever
```

Comment:
We could lower the top_k parameter here to drastically speed up inference. Default is 10, which means the reader's inference method runs on 10 documents. We could set it to 2, which should reduce inference time to a fifth. It will change the predicted answers. Maybe you could try it out while we're at it but we're not too concerned about the runtime of e2e tests anyway.
I saw that the pipeline.run method using the pipeline defined here sets `top_k` to 10. So in case you do change the yaml you'll also need to remove the parameter from the run method.

```yaml
params:
  top_k: 2
```